app/main.py

```diff
@@ -140,20 +140,36 @@ async def swagger_ui_redirect():
 for router in routers_to_include:
     app.include_router(router)
 
+json_data_loader.load_to_database(next(get_db()))
+
 
-# TODO: I add the quote day to the home page
-# until the relevant calendar view will be developed.
 @app.get("/", include_in_schema=False)
 @logger.catch()
-async def home(request: Request, db: Session = Depends(get_db)):
-    quote = daily_quotes.get_quote_of_day(db)
+async def home(
+    request: Request,
+    db: Session = Depends(get_db),
+) -> templates.TemplateResponse:
+    """Home page for the website."""
+    user_id = False
+    if "Authorization" in request.cookies:
+        jwt = request.cookies["Authorization"]
+        user = await current_user(request=request, db=db, jwt=jwt)
+        user_id = user.user_id
+    is_connected = bool(user_id)
+    quote_of_day = daily_quotes.get_quote_of_day(db)
+    if is_connected and daily_quotes.is_quote_favorite(
+        db,
+        user_id,
+        quote_of_day,
+    ):
```

Comment:
Export to variable in order to  make the reading more fluent:
`quote = daily_quotes.is_quote_favorite(db, user_id, quote_of_day)`